setup.py

```diff
@@ -74,5 +74,5 @@ def run(self):
     include_package_data=True,
     # setup_requires=['pyyaml', 'numpy', 'matplotlib', 'aacgmv2'],
     install_requires=['pyyaml', 'numpy', 'matplotlib', 'aacgmv2',
-                      'pydarnio'],
+                      'pydarnio', 'cmocean'],
```

Comment:
See earlier comment about not requiring more things than necessary. One of the matplotlib colormaps should be okay for our purpose.